src/request.rs

```diff
@@ -46,13 +45,14 @@ pub trait KvRequest: Sync + Send + 'static + Sized {
         self,
         pd_client: Arc<impl PdClient>,
     ) -> BoxStream<'static, Result<Self::RpcResponse>> {
-        self.retry_response_stream(pd_client, DEFAULT_REGION_BACKOFF)
+        self.retry_response_stream(pd_client, DEFAULT_REGION_BACKOFF, DEFAULT_LOCK_BACKOFF)
```

Comment:
Pessimistic lock and optimistic lock has different backoffs. When doing pessimistic locking, no extra waiting is needed at the client because it also waits at TiKV. But we need to wait at the client for optimistic locks.